src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -63,7 +65,17 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP)
   if (document.HasMember("id") && document.HasMember("idPattern"))
   {
     LM_W(("Bad Input (both 'id' and 'idPattern' specified"));
-    eP->errorCode.fill("JSON Parse Error III", "both 'id'and 'idPattern'specified");
+    eP->errorCode.fill("JSON Parse Error", "both 'id' and 'idPattern' specified");
```

Comment:
Not sure if I understand in which operations this code is executed, but in "POST /v2/entities" operation `idPattern` is forbidden in any case (no mather if in combinations with `id` or not).